DataFormats/Detectors/FIT/FT0/include/DataFormatsFT0/ChannelData.h

```diff
@@ -0,0 +1,48 @@
+// Copyright CERN and copyright holders of ALICE O2. This software is
+// distributed under the terms of the GNU General Public License v3 (GPL
+// Version 3), copied verbatim in the file "COPYING".
+//
+// See http://alice-o2.web.cern.ch/license for full licensing information.
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+/// \file ChannelData.h
+/// \brief Class to describe fired and  stored channels for the BC and to refer to channel data
+/// \author Alla.Maevskaya@cern.ch
+
+#ifndef _FT0_CHANNELDATA_H_
+#define _FT0_CHANNELDATA_H_
+
+#include <Rtypes.h>
+
+namespace o2
+{
+namespace ft0
+{
+struct ChannelData {
+
+  //public:
+
+  int ChId = -1;     //channel Id
+  int CFDTime = -1;  //time in #CFD channels, 0 at the LHC clk center
+  int QTCAmpl = -1;  // Amplitude #channels
+  int ChainQTC = -1; //QTC chain
+
```

Comment:
@AllaMaevskaya  Could you please remind the range of these data members? Do you really need 4 ints?

Reply:
You can simply use 
````
uint8_t ChanID
int8_t ChainQTC
int16_t CFDTime
int16_t QTCAmpl
````
will take the same 8 bytes (instead of current 16) and w/o bit operations.

Reply:
done, thank you